### PR TITLE
LPS-94709 frontend-js-web - Modernize formatXML utility

### PR DESCRIPTION
**Pull request by wincent**
This is revision 3.

Revision 2 was: https://github.com/wincent/liferay-portal/pull/35

Revision 1 was: https://github.com/wincent/liferay-portal/pull/26

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

File: modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_xml.es.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			'<!DOCTYPE note>\r\n' +
+			'<a:note>\r\n' +
+			'\t<a:to>Foo</a:to>\r\n' +
+			'\t<a:from>Bar</a:from>\r\n' +
+			'\t<a:heading>FooBar</a:heading>\r\n' +
+			'\t<a:body>FooBarBaz!</a:body>\r\n' +
+			'</a:note>';
+
+		expect(formatXML(input)).toEqual(expectedOutput);
+	});
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_xml.es.js]: Lovely!

**Comment by wincent** [comment]
Forwarded upstream: https://github.com/brianchandotcom/liferay-portal/pull/74402

**Comment by kresimir-coko** [comment]
I like this fast iteration approach even if it's difficult to evaluate for our sprints but delivers a better product in the end 👍

**Pull request closed without merging**